examples/SmartRefrigerator/ExplorerKitInstructions.md

```diff
@@ -1,247 +1,260 @@
+
+And if you want to run the board from inside a fridge:
 
-  - 3 AA Batteries
+- 3 AA Batteries
 
-### Step 2: Setup the Electric Imp hardware
+### Step 2: Set up the Electric Imp Hardware
 
-First we will need to assemble the impExplorer Kit.  The kit comes with (1) the imp001 card, which has a WiFi radio and microcontroller which drives all the logic for the board and (2) the impExplorer Kit into which the card is plugged.  The impExplorer Kit provides a set of sensors and peripherals which are ready to use. This project will take readings from temperature, humidity and light sensors to determine the current state of your fridge.
+First we will need to assemble the impExplorer Kit. The kit comes with:
+
+1. An imp001 card, which has a WiFi radio and a microcontroller that drives all the logic for the board
+2. The impExplorer board into which the card is plugged.
+
+The impExplorer Kit provides a set of sensors and peripherals which are ready to use. This project will take readings from temperature, humidity and light sensors to determine the current state of your fridge.
 
 #### Hardware Setup
- - Plug the imp001 card into the card slot on the impExplorer Kit
- - Power up your impExplorer Kit with the provided mini-B USB cable or the AA Batteries
- - The imp001 should now have power and be blinking amber/red
 
-Assmbled it should look like this:
+- Plug the imp001 card into the card slot on the impExplorer 
+- Power up your impExplorer Kit with the provided mini-B USB cable or the AA Batteries
+- The imp001 should now have power and be blinking amber (or red if it has been used before)
+
+Assembled, it should look like this:
 
 ![Explorer Kit](http://i.imgur.com/6JssX74.png)
 
-#### Electric Imp BlinkUp
+#### Electric Imp BlinkUp&trade;
 
 Use the Electric Imp mobile app to BlinkUp your device
 
- - Log into your Electric Imp account
- - Enter your WiFi credentials
- - Follow the instructions in the app to [BlinkUp](https://electricimp.com/platform/blinkup/) your device
- - Make a note of the Device ID of your device (the app shows it after successful blink up). You will need it to assign the device in the Electric Imp IDE.
+- Log into your Electric Imp account
+- Tap ‘Configure a Device’
+- Enter your WiFi credentials
+- Follow the instructions in the app to activate your device
+- Make a note of the Device ID of your device (the app shows it after successful BlinkUp). You will need this in Electric Imp’s impCentral&trade;.
 
- If you have any issues getting started with your Electric Imp account or device, see [the full getting started guide](https://electricimp.com/docs/gettingstarted/explorer/quickstartguide/).
+If you have any issues getting started with your Electric Imp account or device, see [the full getting started guide](https://electricimp.com/docs/gettingstarted/explorer/quickstartguide/).
```

Comment:
Probably the [the full getting started guide] link should be changed to https://electricimp.com/docs/tools/impcentral/impcentralintroduction/